query-engine/connectors/mongodb-query-connector/src/root_queries/mod.rs

```diff
@@ -69,7 +71,11 @@ where
     let span = info_span!(
         "prisma:engine:db_query",
         user_facing = true,
-        "db.statement" = %Arc::clone(&query_string)
+        "db.system" = SYSTEM_NAME,
+        "db.statement" = %Arc::clone(&query_string),
+        "db.collection.name" = %builder.collection().unwrap_or_else(|| "raw"),
```

Comment:
I think we shouldn't add it if it doesn't exist, there's no special "raw" collection in the spec.

I now see that it's actually "conditionally required", I misread it as just "required".